src/main/java/nextstep/ladder/doamin/fcatory/LineFactory.java

```diff
@@ -0,0 +1,31 @@
+package nextstep.ladder.doamin.fcatory;
+
+import nextstep.ladder.doamin.Line;
+import nextstep.ladder.strategy.LineStrategy;
+import nextstep.ladder.utils.Preconditions;
+
+import java.util.List;
+import java.util.stream.Collectors;
+import java.util.stream.IntStream;
+
+public class LineFactory {
+    private static final Integer RANGE_START = 1;
+    private static final Integer MINIMUM_HEIGHT_SIZE = 1;
+    private static final Integer MINIMUM_PERSON_SIZE = 2;
+
+    public List<Line> createLines(LineStrategy lineStrategy, Integer countOfPerson, Integer heightOfLadder) {
+        Preconditions.checkMinimumSize(countOfPerson, MINIMUM_PERSON_SIZE,
+                                       String.format("사람 수는 %s 이상 이어야 합니다.", MINIMUM_PERSON_SIZE));
+        Preconditions.checkMinimumSize(heightOfLadder, MINIMUM_HEIGHT_SIZE,
+                                       String.format("사다리 높이는 %s 이상 이어야 합니다.", MINIMUM_HEIGHT_SIZE));
+
+        return IntStream.rangeClosed(RANGE_START, heightOfLadder)
+                .boxed()
+                .map(h -> createLine(lineStrategy, countOfPerson))
```

Comment:
```suggestion
                .map(h -> Line.from(lineStrategy.createPoints(countOfPerson)))
```

메서드를 분리하는 것도 좋지만 생성자를 통해 생성하는 부분에 있어서 별 다른 로직이 없다면 이렇게 사용해도 좋을 것 같아요 :)

Reply:
분리가 좀 과했던것 같네요. 피드백 주신 부분이 좀더 간결 한것 같아서 피드백 반영하도록 하겠습니다! 😄